test/test_run_analysis.rb

```diff
@@ -50,6 +50,88 @@ def _assert_and_puts(output, msg, expect_error = true)
     end
   end
 
+  def _verify_outputs(cli_output_log, testing = false)
+    # Check cli_output.log warnings
+    File.readlines(cli_output_log).each do |message|
+      next if message.strip.empty?
+      next if message.include?('Building ID:')
+      next if message.include?('[openstudio.measure.OSRunner] <1> Cannot find current Workflow Step')
+      next if message.include?('[openstudio.model.Surface] <1> Initial area of other surface')
+      next if _expected_warning_message(message, 'No valid weather file defined in either the osm or osw.')
+      next if _expected_warning_message(message, 'The model contains existing objects and is being reset.')
+      next if _expected_warning_message(message, 'HVAC setpoints have been automatically adjusted to prevent periods where the heating setpoint is greater than the cooling setpoint.')
+      next if _expected_warning_message(message, 'It is not possible to eliminate all HVAC energy use (e.g. crankcase/defrost energy) in EnergyPlus during an unavailable period.')
+      next if _expected_warning_message(message, 'It is not possible to eliminate all water heater energy use (e.g. parasitics) in EnergyPlus during an unavailable period.')
+      next if _expected_warning_message(message, 'No space heating specified, the model will not include space heating energy use. [context: /HPXML/Building/BuildingDetails]')
+      next if _expected_warning_message(message, 'No space cooling specified, the model will not include space cooling energy use. [context: /HPXML/Building/BuildingDetails]')
+      next if _expected_warning_message(message, 'No clothes washer specified, the model will not include clothes washer energy use. [context: /HPXML/Building/BuildingDetails]')
+      next if _expected_warning_message(message, 'No clothes dryer specified, the model will not include clothes dryer energy use. [context: /HPXML/Building/BuildingDetails]')
+      next if _expected_warning_message(message, 'No dishwasher specified, the model will not include dishwasher energy use. [context: /HPXML/Building/BuildingDetails]')
+      next if _expected_warning_message(message, 'No refrigerator specified, the model will not include refrigerator energy use. [context: /HPXML/Building/BuildingDetails]')
+      next if _expected_warning_message(message, 'No cooking range specified, the model will not include cooking range/oven energy use. [context: /HPXML/Building/BuildingDetails]')
+      next if _expected_warning_message(message, 'Home with unconditioned basement/crawlspace foundation type has both foundation wall insulation and floor insulation.')
```

Comment:
I hope this isn't occurring in a baseline run. Though it's pretty weird even for an upgrade. Seems like it should be reviewed.

Reply:
I double-checked this. It doesn't occur in national baseline. It does occur in testing baseline.